tst/testinstall/evalstring.tst

```diff
@@ -0,0 +1,24 @@
+gap> START_TEST("evalstring.tst");
+
+#
+gap> EvalString("");
+
+#
+gap> f := function() local x; return x; end;;
+gap> EvalString("f();");
+Error, Variable: 'x' must have an assigned value
+Error, Could not evaluate string.
+
+
+#
+gap> EvalString("x := 15;");
```

Comment:
This actually fails with the old `EvalString`. Perhaps this change in behavior is intentional, but I did no see any mention of it before. Same for the next example -- the old `EvalString` expected the first "statement" in the string to be an expression, as can be guessed from the name `expr` given to its argument string.

Reply:
(`expr` is used in the documentation, at least, though apparently not in the implementation)